models/game.rb

```diff
@@ -127,6 +127,7 @@ def to_h(include_actions: false, player: nil)
       description: description,
       user: user.to_h,
       players: ordered_players.map(&:to_h),
+      min_players: min_players || nil,
```

Comment:
why do you need an || nil

Reply:
I am not sure if it is necessary, but I was worried about games created before this property was added.

Reply:
no it's not necessary